plugin/src/generators/create-event/generator.ts

```diff
@@ -0,0 +1,113 @@
+import { Tree, logger, readJson, updateJson } from '@nx/devkit';
+import { CreateEventGeneratorSchema } from './schema';
+import { exit } from 'node:process';
+import { Community } from '../models/community';
+
+const COMMUNITIES_PATH = 'angular-hub/src/public/assets/data/community.json';
+
+export async function createEventGenerator(
+  tree: Tree,
+  options: CreateEventGeneratorSchema,
+) {
+  const packageJson = readJson(tree, 'package.json');
+  const angularCorePackage = packageJson['dependencies']['@angular/core'];
```

Comment:
angular/core check to be removed as the generator is always used in the workspace.